<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PromptWindow</class>
 <widget class="QMainWindow" name="PromptWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Procesador de Texto Técnico</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>2</number>
      </property>
      <widget class="QWidget" name="leftWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="labelTitulo">
          <property name="styleSheet">
           <string notr="true">font-size: 16px; font-weight: bold;</string>
          </property>
          <property name="text">
           <string>Ingresá un texto con tareas técnicas</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="inputPrompt">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>200</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>200</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 14px;</string>
          </property>
          <property name="placeholderText">
           <string>Ej: Crear interfaz de usuario, conectar con backend, diseñar base de datos...</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QPushButton" name="btnProcesar">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">
                      padding: 8px;
                      font-weight: bold;
                      background-color: #4CAF50;
                      color: white;
                    </string>
          </property>
          <property name="text">
           <string>Procesar texto</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="resultadoTexto">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>200</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>200</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 14px;</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Aquí se mostrará la lista de tareas...</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QPushButton" name="btnExportar">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">
                      padding: 8px;
                      font-weight: bold;
                      background-color: #2196F3;
                      color: white;
                    </string>
          </property>
          <property name="text">
           <string>Exportar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="rightWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="labelHistorial">
          <property name="styleSheet">
           <string notr="true">font-size: 16px; font-weight: bold; padding-bottom: 10px;</string>
          </property>
          <property name="text">
           <string>Historial de consultas</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tablaHistorial">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="textElideMode">
           <enum>Qt::ElideNone</enum>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
